src/wp-includes/media.php

```diff
@@ -1843,6 +1843,9 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			// Use alternate mime types when specified and available.
+			$filtered_image = wp_image_use_alternate_mime_types( $filtered_image, $attachment_id );
```

Comment:
This needs to be wrapped in a `$attachment_id > 0` check like the other function calls above.

Reply:
ah, good catch. when is attachment id 0 here?

Reply:
wrapped, question still open

Reply:
@adamsilverstein It would be 0 if the image is not an attachment. You could just include an image from another website for example, then there is no attachment, so the attachment ID is 0.

In that case we can still do basic things like for example add the `loading` attribute, but we can't do things that require image metadata to be present.